crates/ide-completion/src/render/variant.rs

```diff
@@ -94,3 +94,12 @@ pub(crate) fn format_literal_label(name: &str, kind: StructKind) -> SmolStr {
         StructKind::Unit => name.into(),
     }
 }
+
+/// Format a struct, etc. literal option for lookup used in completions filtering.
+pub(crate) fn format_literal_lookup(name: &str, kind: StructKind) -> SmolStr {
```

Comment:
`()` and `{}` are still needed as current testing infra does not allow duplicated lookup. https://github.com/rust-lang/rust-analyzer/blob/ec3586eab943d7c6c83d211f629a691fc8c14603/crates/ide-completion/src/tests.rs#L186-L190

I guess IDEs will happily accept duplicated lookup? Anyway these extra parentheses and brackets also allow completing something like `E{` so I'd prefer to keep them here.